README.md

```diff
@@ -121,16 +137,16 @@ not required):
        "irods_default_resource" : "demoResc"
    }
    ```
-   Note that the *"irods_host"* and *"irods_port"* identify the iRODS server that the DSI will contact during each request.
+   Note that the *"irods_host"* and *"irods_port"* identify the iRODS server that the DSI will contact during each request. Be sure to set the *irods_default_resource*, this variable is not set when you create the file with *iinit* or when you copy it over from another user.
    
 2. As the user who runs the GridFTP server, try an `ils` icommand to verify that 
    the information set in the *irods_environment.json* are fine. If needed, perform 
    an `iinit` to authenticate the iRODS user. 
 
-2. Add the following lines to the GridFTP configuration file ( tipically 
-   *$GLOBUS_LOCATION/etc/gridftp.conf* ):
+3. Add the following lines to the GridFTP configuration file (typically 
+   *$GLOBUS_LOCATION/etc/gridftp.conf*):
    ```
-   $LD_LIBRARY_PATH "$LD_LIBRARY_PATH:/preferred_path/iRODS_DSI"
+   $LD_LIBRARY_PATH "$LD_LIBRARY_PATH:/preferred_path/iRODS_DSI/B2STAGE-GridFTP/"
```

Comment:
`make install` copies the .so file from the build directory into the `iRODS_DSI` destination folder - and that's where the GridFTP server should be loading them from....